src/mongocxx/events/heartbeat_failed_event.cpp

```diff
@@ -49,6 +49,11 @@ std::uint16_t heartbeat_failed_event::port() const {
         ->port;
 }
 
+bool heartbeat_failed_event::awaited() const {
+    return libmongoc::apm_server_heartbeat_failed_get_awaited(
+        static_cast<const mongoc_apm_server_heartbeat_failed_t*>(_failed_event));
```

Comment:
spacing here looks different from other methods, clang format?

Reply:
I ran clang format again to check, but that is how it formats it. I split these into separate lines for readability.